authors/apps/authentication/views.py

```diff
@@ -46,7 +46,7 @@ def post(self, request):
         return Response(serializer.data, status=status.HTTP_200_OK)
 
 
-class UserRetrieveUpdateAPIView(RetrieveUpdateAPIView):
+class UserRetrieveUpdateAPIView(GenericAPIView):
```

Comment:
@dorothyas I think this should be `GenericRetrieveUpdateAPIView` in order not to change the functionality of the endpoint.

Reply:
okay